### DISCUSSION TITLE
Timeline Zoom Level below 1 to see 24h time span

### DISCUSSION THREAD

**Discussion by ManifestTheData** [ivan-lednev/obsidian-day-planner#574] [Features & bugs]
Hello everyone :)

I would like to be able to set the Timeline Zoom Level to below 1. Currently (at max. vertical size) you can see about 12.5 hours of your day. However, it would be nice to see more than that (e.g., the entire 24h + some unscheduled/all day tasks).
My goal is to have the plugin run in my right sidebar over about half the vertical size and/but still be able to see the events/tasks of the entire day.

Thanks a lot in advance, and thank you for your work! Really loving the plugin! :pray: :+1: 

P.S. I'm hoping this is the right section. First time GitHub user. :sweat_smile: 

Kind regards,
MtD

**Comment by ivan-lednev** [comment]
This is not the right section. Here is the template: https://github.com/ivan-lednev/obsidian-day-planner/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yaml
